roles/lib_openshift/src/ansible/oc_serviceaccount_secret.py

```diff
@@ -0,0 +1,29 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module to manage service account secrets.
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            state=dict(default='present', type='str',
+                       choices=['present', 'absent', 'list']),
+            debug=dict(default=False, type='bool'),
+            namespace=dict(default=None, required=True, type='str'),
```

Comment:
Doc says namespace = 'default' by default.  Consider removing 'required=True' and instead setting default='default' to match doc.